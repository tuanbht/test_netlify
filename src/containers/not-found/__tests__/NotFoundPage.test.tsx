import { shallow } from 'enzyme';
import NotFoundPage from '../index';
import React from 'react';

describe('NotFoundPage', () => {
  it('renders template exactly', () => {
    expect(shallow(<NotFoundPage />)).toMatchSnapshot();
  });
});
