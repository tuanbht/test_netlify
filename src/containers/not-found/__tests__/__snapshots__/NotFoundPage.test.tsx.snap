// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage renders template exactly 1`] = `
<div
  id="mobile-screen"
>
  <h1
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    404 Not Found!
  </h1>
</div>
`;
